src/lava_callback.py

```diff
@@ -211,6 +228,16 @@ def async_job_submit(api_helper, node_id, job_callback):
     job_result = job_callback.get_job_status()
     device_id = job_callback.get_device_id()
     storage_config_name = job_callback.get_meta('storage_config_name')
+
+    if job_actions := job_callback.get_job_definition('actions'):
+        if deploy_images := job_actions[0].get('deploy', {}).get('images'):
+            pass
+        else:
+            # Some deployment types does not have 'images' key
+            deploy_images = job_actions[0].get('deploy', {})
```

Comment:
Let's avoid a `pass` and reduce duplication

```suggestion
        # NOTE: Some deployment types do not have 'images' key
        deploy_images = job_actions[0].get('deploy', {})
        if 'images' in deploy_images:
            deploy_images = deploy_images['images']
```